bindings.c

```diff
@@ -4449,18 +4455,20 @@ static void __attribute__((constructor)) collect_and_mount_subsystems(void)
 	char cwd[MAXPATHLEN];
 	size_t len = 0;
 	int i, init_ns = -1;
+	bool found_unified = false;
 
 	if ((f = fopen("/proc/self/cgroup", "r")) == NULL) {
 		lxcfs_error("Error opening /proc/self/cgroup: %s\n", strerror(errno));
 		return;
 	}
 
 	while (getline(&line, &len, f) != -1) {
-		char *p, *p2;
+		char *idx, *p, *p2;
 
 		p = strchr(line, ':');
 		if (!p)
 			goto out;
+		idx = line;
```

Comment:
This might be clearer so might be worth it anyway, but just to check - is there any reason you couldn't just use 'line' instead of idx in the strcmp below?